src/Instance/MoveItemsContext/MoveItemsContext.js

```diff
@@ -85,6 +85,8 @@ const MoveItemsContext = ({ children, moveItems }) => {
           selectItemsForDrag,
           isItemsDragSelected,
           getDraggingItems,
+          selectedItemsMap,
+          onSelect : onDragEnd,
```

Comment:
select i not drag, drag and select should invoke common code and not mix everything in one function